experimental/devices/bh1750/bh1750.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package bh1750
+
+import (
+	"encoding/binary"
+	"time"
+
+	"periph.io/x/periph/conn/i2c"
+	"periph.io/x/periph/conn/physic"
+)
+
+// I2CAddr i2c default address.
+const I2CAddr uint16 = 0x23
+
+// AlternativeI2CAddr i2c alternative address.
+const AlternativeI2CAddr uint16 = 0x5c
+
+// Resolution represents the measurements modes.
```

Comment:
In [bmxx80](https://periph.io/x/periph/devices/bmxx80), I was able to hide this by having the client either use:
- Use [SenseContinous()](https://periph.io/x/periph/conn/physic#SenseEnv) for continuous sensing.
- Use Sense() for one off operation.

Then you can expose the resolutions 0.5lx, 1lx, and 4lx and then have the code underneath use the right mode based on the function used.

https://github.com/google/periph/blob/master/devices/bmxx80/bmxx80.go#L431 shows how it's implemented.